lib/Makefile

```diff
@@ -12,12 +12,18 @@ $(OBJECTS): %$(OBJEXT): %.c %.h
 clean-l:
 	rm -f $(OBJECTS)
 
+ifeq "$(FHS_ENABLED)" "yes"
```

Comment:
`DESTDIR` should never be prepended here, since WEBAPP_DIR is later used only as contents inside files, not for actual paths that things are installed to. See for an explanation https://www.gnu.org/prep/standards/html_node/DESTDIR.html